docker-compose.yml

```diff
@@ -0,0 +1,29 @@
+name: a12n-server-dev
+
+services:
+  db:
+      image: mysql:8.0
+      restart: always
+      ports:
+        - "3306:3306"
+      environment:
+        - DB_HOST=db
+        - DB_PASSWORD='password'
+        - DB_USER=a12nserver
+        - DB_DATABASE=a12nserver
+  a12n-server:
+    build: 
+      context: .
+      dockerfile: Dockerfile
+    labels:
+      - org.opencontainers.image.source=https://github.com/curveball/a12n-server
```

Comment:
Do you know if/why this is needed? I assume we just want to build from local by default.